tests/testthat/test-5prevalence.R

```diff
@@ -62,6 +62,36 @@ test_that("getPrevalence", {
     pr2 <- getPrevalence(gp_null, detection=0.004, as_relative=TRUE, rank = "Family") 
     expect_equal(pr1, pr2)
     
+    # Check that na.rm works correctly
+    tse <- GlobalPatterns
+    # Get reference value
+    ref <- getPrevalence(tse, assay.type = "counts")
+    # Add NA values to matrix
+    remove <- c(1, 3, 10)
+    assay(tse, "counts")[remove, ] <- NA
+    # There should be 3 NA values if na.rm = FALSE. Otherwise there should be 0
+    expect_true( sum(is.na(
+        getPrevalence(tse, assay.type = "counts", na.rm = FALSE) )) == 3)
```

Comment:
does this also work when `rank` argument is specified? Because earlier there were differences how this works when rank is additionally provided, compared to this present example.

Reply:
These modifications do not affect this thing, I guess. (I am not sure what differences there were)

When rank is specified

1. TreeSE is agglomerated to level specified by `rank`. `agg.na.rm` is used to specify whether to drop those rows that do not have taxonomy in level specified by `rank.` --> Instead of "phylum_something" & "class_something2" there are only taxa that have taxonomy info in class level, for instance.

2. If there are NA values, agglomerated tables have also NA values in higher level rank --> This might be misleading? We could add parameter to agglomerateByRank that removes NAs? (agglomerateByRanks --> mergeRows --> .merge_rows --> scuttle::sumCountsAcrossFeatures)

3. getPrevalence calculates prevalence of features. With `na.rm,` user can specify whether to remove NA values.


So currently, when there are NA values and rank is specified, there are some taxa that have prevalence NA.